consultation_analyser/factories.py

```diff
@@ -232,6 +232,8 @@ def answers(consultation_response, creation_strategy, value, **kwargs):
 
 class ProcessingRunFactory(factory.django.DjangoModelFactory):
     consultation = factory.SubFactory(ConsultationFactory)
+    started_at = timezone.now()
+    finished_at = timezone.now() + timezone.timedelta(minutes=1)
```

Comment:
Finishing in the future by default is a bit weird, maybe we should start in the past instead?

Reply:
Changed to `started_at = timezone.now() - timezone.timedelta(minutes=1)` and finished at `timezone.now()`.